tests/lib/bin/mountinfo-tool

```diff
@@ -911,34 +913,44 @@ class RenumberMountOptionsTests(unittest.TestCase):
         """
         numbers are allocated from subsets matching the key, this reduces delta.
         """
-        self.assertEqual(renumber_mount_option("size=100", self.seen), "size=0")
-        self.assertEqual(renumber_mount_option("size=100", self.seen), "size=0")
-        self.assertEqual(renumber_mount_option("size=200", self.seen), "size=1")
-        self.assertEqual(renumber_mount_option("size=100", self.seen), "size=0")
+        self.assertEqual(renumber_mount_option("pipe_ino=100", self.seen), "pipe_ino=0")
```

Comment:
so just to understand this change here, previously all the size options were rewritten in order of when they were seen, i.e. the first size option seen becomes size=0 and the next unique value becomes size=1, etc. 

And now, this specific test no longer tests the size parameter and instead tests that the pipe_ino parameter gets rewritten like that. 

The new size behavior is tested below with `test_size_not_allocated`.

Reply:
I moved the test for the size parameter to a new function, replacing size with another renumbered quantity, pipie_ino.